### PR TITLE
Monads removal (`funcool.cats`); interceptors behaviour change.

### PR DESCRIPTION
**Pull request by jehaby**
- [x] removed monads (`funcool.cats` lib) 
- [x] changed interceptors behaviour
- [x] added https://github.com/Flexiana/framework/blob/rid-of-cats-2/CHANGELOG.md

For discussion on monads & interceptors check out zulip: https://chat.flexiana.com/#narrow/stream/89-open-source/topic/xiana.20interceptors/near/458279

There's also https://github.com/Flexiana/framework/projects/4 with the next tasks. Feel free to expand/edit it.

### Tester info
1) do `lein install` from the project root
2) run `lein test`, `./example-tests.sh`

### CODE REVIEW

File: examples/cli-chat/src/shared/config.clj [hunk 1]
```diff
@@ -1 +1 @@
-(ns config)
+
```
[the remark this answers is not in the document]
Comment (jehaby) [reply examples/cli-chat/src/shared/config.clj]: fixed https://github.com/Flexiana/framework/pull/217/commits/5ff16002332652041c1f9d1423e7c7c59c90f175

File: deps.edn [hunk 2]
```diff
@@ -0,0 +1,26 @@
+{:paths ["src"]
+ :deps {com.draines/postal                      {:mvn/version "2.0.5"},
+        com.flexiana/tiny-rbac                  {:mvn/version "0.1.1"},
+        com.taoensso/timbre                     {:mvn/version "5.1.2"},
+        crypto-password/crypto-password         {:mvn/version "0.3.0"},
+        funcool/cuerdas                         {:mvn/version "2.2.1"},
+        info.sunng/ring-jetty9-jadapter          {:mvn/version "0.17.4"},
```
[the remark this answers is not in the document]
Comment (jehaby) [reply deps.edn]: fixed https://github.com/Flexiana/framework/pull/217/commits/5ff16002332652041c1f9d1423e7c7c59c90f175

File: src/xiana/interceptor.clj [hunk 3]
```diff
@@ -46,7 +35,7 @@
   procedure, if none was found execute: `xiana/ok`."
```
Comment (soulflyer) [inline src/xiana/interceptor.clj]: docstring doesn't match behaviour

**Review by soulflyer — APPROVED** [review]
Looks good to me.

**Pull request merged**